src/UI/RadioButton.elm

```diff
@@ -0,0 +1,96 @@
+module UI.RadioButton exposing (radioGroup)
```

Comment:
Apparently, this is called only `radio` everywhere, ( I checked all the places). so, I would rename it to `UI.radio`